frontend/src/pages/logInPage.js

```diff
@@ -1,6 +1,5 @@
 import React, { useState } from 'react';
 
-
 const LogInPage = () => {
```

Comment:
It looks like the filename here is still logInPage.js which is causing an error when I try to run the app.

Reply:
I got the same error and fixed it before I submitted it. Nelly also fixed it in her PR. I am not sure how this keeps happening. I'll try again.

Reply:
Fixed, I hope!